jdaviz/configs/default/plugins/plot_options/tests/test_plot_options.py

```diff
@@ -25,24 +25,23 @@ def test_multiselect(cubeviz_helper, spectrum1d_cube):
     for viewer_name in ['flux-viewer', 'uncert-viewer']:
         assert cubeviz_helper.app.get_viewer(viewer_name).state.show_axes is False
 
-    for viewer_name in ['mask-viewer', 'spectrum-viewer']:
-        assert cubeviz_helper.app.get_viewer(viewer_name).state.show_axes is True
+    assert cubeviz_helper.app.get_viewer('spectrum-viewer').state.show_axes is True
 
     assert po.show_axes.sync['mixed'] is False
 
     # adding another viewer should show a mixed-state
-    po.viewer.selected = ['flux-viewer', 'uncert-viewer', 'mask-viewer', 'spectrum-viewer']
+    po.viewer.selected = ['flux-viewer', 'uncert-viewer', 'spectrum-viewer']
     # spectrum viewer is excluded from the show_axes logic
-    assert len(po.show_axes.linked_states) == 3
-    assert len(po.show_axes.sync['icons']) == 3
-    assert po.show_axes.sync['mixed'] is True
+    assert len(po.show_axes.linked_states) == 2
+    assert len(po.show_axes.sync['icons']) == 2
+    assert po.show_axes.sync['mixed'] is False
```

Comment:
we could keep this as True and still cover the mixed state intention of this test by originally only disabling the axes in a single cube viewer instead of both.

Reply:
Not sure I fully grok this point; my review will leave this comment out and defer to Kyle's better judgement